dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/location/internal/Activator.java

```diff
@@ -38,6 +41,8 @@
 public class Activator implements BundleActivator {
     private static final TraceComponent tc = Tr.register(Activator.class);
 
+    private static final String JAVA_CONDITION_ID = "io.openliberty.java.version";
```

Comment:
We should place this condition constant somewhere such that it can be used in by filters annotated in `@SatisfyingConditionTarget`

I suggest a public constant in `com.ibm.ws.kernel.service.util.JavaInfo` that we can javadoc for the Java version condition.